pkgs/development/python-modules/scikitlearn/default.nix

```diff
@@ -1,21 +1,29 @@
 { stdenv, buildPythonPackage, fetchPypi
 , gfortran, glibcLocales
 , numpy, scipy, pytest, pillow
+, cython
+, joblib
 }:
 
 buildPythonPackage rec {
   pname = "scikit-learn";
-  version = "0.20.3";
+  version = "0.21.2";
   # UnboundLocalError: local variable 'message' referenced before assignment
   disabled = stdenv.isi686;  # https://github.com/scikit-learn/scikit-learn/issues/5534
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "c503802a81de18b8b4d40d069f5e363795ee44b1605f38bc104160ca3bfe2c41";
+    sha256 = "1nvj9j16y1hz9gm0qwzpnx2zmz55c63k1fai643migsyll9c7bqa";
   };
 
   buildInputs = [ pillow gfortran glibcLocales ];
-  propagatedBuildInputs = [ numpy scipy numpy.blas ];
+  propagatedBuildInputs = [
+    numpy
+    scipy
+    numpy.blas
+    cython
```

Comment:
I _think_ `cython` can be a `buildInput` instead?

Reply:
You're right, thanks.